client/rpc-api/src/state/mod.rs

```diff
@@ -144,6 +145,10 @@ pub trait StateApi<Hash> {
 		at: Option<Hash>,
 	) -> FutureResult<Vec<StorageChangeSet<Hash>>>;
 
+	/// Returns proof of storage entries at a specific block's state.
+	#[rpc(name = "state_getProof")]
+	fn proof(&self, keys: Vec<StorageKey>, hash: Hash) -> FutureResult<StorageProof>;
```

Comment:
I think we should call it `read_proof` and `state_getReadProof`.

`proof` alone isn't that expressive.

Reply:
I found that the same RPC uses `eth_getProof` as the name in [EIP-1186](https://github.com/ethereum/EIPs/issues/1186). Do we need to be consistent with EIP-1186?

Reply:
We are not developing eth here ;) And I don't think that other RPC's are following any `EIP`.

Reply:
OK， my first thought was to use `state_getReadProof` as well.